libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -339,6 +367,21 @@ void AP_DDS_Client::write_static_transforms()
     }
 }
 
+void AP_DDS_Client::write_local_position_topic()
```

Comment:
Please rename to `write_local_pose_topic` as it's a pose, not a position, since the pose encodes both position and orientation. Same name change applies elsewhere

Reply:
Changed the PR name, the commit name and the variables.